projects/lib/package.json

```diff
@@ -5,11 +5,23 @@
     "@angular/common": ">=12.0.0",
     "@angular/core": ">=12.0.0",
     "@angular/forms": ">=12.0.0",
```

Comment:
should this also be raised on 12, or maybe 13?

Reply:
I do not know. You are the ngProfis 😘

Reply:
will do it :)